src/main/resources/data/computercraft/lua/rom/programs/id.lua

```diff
@@ -13,16 +13,32 @@ if sDrive == nil then
     end
 
 else
+    local bAudio = disk.hasAudio(sDrive)
+    if bAudio then
```

Comment:
```suggestion
    if disk.hasAudio(sDrive) then
```

Reply:
Should i do that with all locals? Cause i will gladly get rid of all of them if we are dropping idea of "educational and stupidly long programs".

Reply:
Well, ones we don't reuse. But yes :).

Reply:
Personal take is that the best way CraftOS programs can be educational is if they are readable and demonstrate good(ish) practice. They don't need to be overly explicit/verbose, as that's what examples are for!